### ISSUE TITLE
Microsoft teams option no showing for issue alerts

### ISSUE THREAD

**Issue by rodolfoBee** [getsentry/sentry#80311]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Install the microsoft teams as per: https://docs.sentry.io/organization/integrations/notification-incidents/msteams/
2. Create a new issue alert
3. Try to add action to send messages to Microsoft Teams integration

Troubleshooting actions taken:
- trying to reinstall the integration, same issue
- trying to refresh list of available integrations

issue was reported in an [internal ticket.](https://sentry.zendesk.com/agent/tickets/137272)

### Expected Result

The option "Send a Microsoft Teams notification" is shown in the list.

### Actual Result

No "Send a Microsoft Teams notification" option:
![Image](https://github.com/user-attachments/assets/b4f9890f-698d-498f-a7a2-a171a5682bc1)

Refreshing the list does not solve the issue, the integration shows as disabled in the HTTP response
```
"label": "Send a notification to the {team} Team to {channel}",
"enabled": false,
```

 The option to send message to MSTeams is available on Metric alerts:
![Image](https://github.com/user-attachments/assets/1ea48407-a1e6-442b-b05d-5c0763f16037)

### Product Area

Alerts

**Comment by rachrwang** [comment]
@sentaur-athena - is this something your team can help take a look at?

**Comment by sentaur-athena** [comment]
Yes this is not an alert issue. There's definitely something going on with the installation and the alert is not showing msteams as option because it shows up as not enabled in the list of integrations. I'll look at this later next week.

**Comment by Kleinkind** [comment]
Any update on this? We are facing the same problem and also dont get it work using a "Custom Measurement"-Alert either (there the Teams option show up but upon saving we get a 500 "Unable to save alert: Internal Error").

**Comment by sentaur-athena** [comment]
Sorry no updates yet. I'll post when we have an update.

**Comment by stefanchopin** [comment]
This issue is nearly two months old.  This is an advertised standard feature of the product . We need this resolved to keep using Sentry. What is the ETA for resolution?

**Comment by iamrajjoshi** [comment]
Hello! Thanks for reaching out! It seems like you completed a personal installation of the MsTeams integration. This installation type doesn't support [issue alerting](https://sentry.zendesk.com/hc/en-us/articles/21412206819355-Why-can-t-I-select-MS-Teams-Tenants-to-notify-on-Alerts). 

If you follow the guide [here](https://docs.sentry.io/organization/integrations/notification-incidents/msteams/), you should be able to set up issue alerts for the MsTeams integration. This guide explains how to integrate an MsTeams team with Sentry.

We realize that this process is extremely confusing and will be working on making this clearer.

Let us know if there is anything else we can do!

**Comment by iamrajjoshi** [comment]
Hey @stefanchopin, thanks for responding! I only see a personal installation. Could you please try setting up a team based one so that we can triage this further?

**Comment by yassine-ah** [comment]
> Hey [@stefanchopin](https://github.com/stefanchopin), thanks for responding! I only see a personal installation. Could you please try setting up a team based one so that we can triage this further?

Hello @iamrajjoshi 

In [Microsoft Teams App Store](https://teams.microsoft.com/l/app/5adee720-30de-4006-a342-d454317db1d4), we don't have the possibility to click the arrow and select "Add to Teams" option.
Is there a workaround to link alerts to a channel ? as we can only install it as personal right now from the MS Teams AppStore.

Thanks

![Image](https://github.com/user-attachments/assets/71b4bcc0-b9de-45e5-9ddf-472658f4420c)

**Comment by sentaur-athena** [comment]
I can repro not seeing the add button. I wonder if it has something to do with your MSTeam org settings. Looking to figure out more.

**Comment by fabian-michael-asi** [comment]
We're facing the same problem. There is no "Add to team" option. Just an "Add" Button. Thought we could select a channel after adding the app and it actually shows up in the "Apps" tab in the desired team we want the notifications to be but it postet the welcome message into the wrong channel of the team. Not the one we selected. In Sentry when trying to create an alert there is no Teams option. In `Settings -> [orga] -> Integrations` the Teams integration is active.

**Comment by sentaur-athena** [comment]
It turns out MsTeams updated this in the UI. Still if you have teams you can get this done by clicking on the app -> choosing the right team channel. 

<img width="573" alt="Image" src="https://github.com/user-attachments/assets/b44ca726-8610-4695-9806-f7248004d7d7" />

And then installation will happen on a team channel. Note that you can choose the team but not the channel within the team at this step. You will receive a welcome message that then you can click on and will take you to Sentry.

<img width="975" alt="Image" src="https://github.com/user-attachments/assets/7f35fbf9-c65e-46c8-98bc-8c3806db845d" />

@fabian-michael-asi I agree with you being confusing especially with the fact that the UI doesn't look like the screenshots in the docs anymore. Will update the docs.

**Comment by sentaur-athena** [comment]
I am going to mark this issue as resolved since the initial issue is resolved. Please create new tickets for other cases.
